### PR TITLE
Bugfix/2838

### LINKED ISSUE #2838: [USERSNAP] Remove `See my proposal` after voting
## Scope

- [ ]  Update CTA to "Back to proposals" and take user to [/proposals](https://dao.joystream.org/#/proposals/current) on click

_________
**Taken at**: https://dao.joystream.org/#/proposals/preview/81
**Fields**:
- Houston, we have a problem.: Remove `See my proposal` after voting
- Comment: This modal is not needed because user is most likely on the proposal page after voting
- see above
- Feedback: l1dev

SCREENSHOT
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/854ed8c4-0b08-4433-a5df-7ecbe7eb4027-annotated_06750e58-d020-46e6-8ece-0ea6b5101f17.png?etag=a83c9bcc0b6bc4c7dbbc67d3e636061a)*

**Browser**: Chrome 99 (Linux)
**Screen size**: 1920x1080
**Browser size**: 2868x1460
**[Open #198 in Usersnap](https://app.usersnap.com/l/feedback/9929af09-49d1-446c-a63e-bace8ae880e7?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

### PR DESCRIPTION
**Pull request by singulart**
Fix #2838

### CODE REVIEW

File: packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx [hunk 1]
```diff
@@ -35,8 +35,8 @@ export const SuccessModal = ({ onClose, proposalId, proposalType, proposalTitle
         </Info>
       </ModalBody>
       <ModalFooter>
-        <ButtonGhost size="medium" onClick={openProposal}>
-          See my Proposal
+        <ButtonGhost size="medium" onClick={openProposals}>
+          Back to proposals
```
Comment (thesan) [inline packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx:39]: Thank you for the PR it is exactly what is needed for the "VoteForProposal" modal. But #2838 is only targeted at this modal not at the "AddNewProposal" modal. We do want modal creators to have a direct link to the newly created modal.

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx (hunk 1 above)
Comment (singulart) [reply packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx:39]: Apologies, I don't get your comment. I cannot find "See my Proposal" anywhere else in the codebase, including `VoteForProposalModal.ts`

Comment (thesan) [reply packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx:39]: Yes

Comment (singulart) [reply packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx]: Fixed

**Review by thesan — APPROVED** [review]
Thank you for the PR. LGTM !

**Pull request merged**